tests/data/aws/ec2/internet_gateways.py

```diff
@@ -0,0 +1,13 @@
+TEST_INTERNET_GATEWAYS = [
+    {
+        'Attachments': [{'State': 'available', 'VpcId': 'vpc-038cf'}],
+        'InternetGatewayId': 'igw-013cb',
+        'OwnerId': '12345',
+        'Tags': [],
+    },
```

Comment:
Is it worth adding some tests where the IGWs are in states besides available?

Reply:
Discussed in person - we will ingest the igw regardless of states and we will store the state on the node